zerver/migrations/0001_initial.py

```diff
@@ -47,8 +53,8 @@ class Migration(migrations.Migration):
 CREATE TRIGGER zerver_message_update_search_tsvector_async
   BEFORE INSERT OR UPDATE OF subject, rendered_content ON zerver_message
   FOR EACH ROW EXECUTE PROCEDURE append_to_fts_update_log();
-"""),
-            ]
+""")
+        zulip_postgres_migrations = [full_text_search_migration]
 
```

Comment:
Thanks for deciding them. It's reasonable.

I've put `scripts/setup/enable-pgroonga` and `scripts/setup/disable-pgroonga` and removed related codes from `tools/postgres-init-deb-db` and `zerver/migrations/0001_initial.py`. `docs/full-text-search.md` is updated.